pkg/kubectl/resource/builder_test.go

```diff
@@ -234,32 +234,73 @@ func TestNodeBuilder(t *testing.T) {
 }
 
 func TestPathBuilderWithMultiple(t *testing.T) {
+	// This builder *should not* error
```

Comment:
This test would be easier to read if it followed the 
```go
tests := []struct{
    name string
    directory string
    expectedNames []string
} {
    {"recursive-pod", "../.../../examples/recursive/pod", []string{"busybox-1", "busybox-2"}},
}

// iterate over tests, run each one separately, and verify results spearately
```

Right now this is a bit of an eye catcher.